### ISSUE TITLE
Incorrect value on iostat wMB/s when using JSON output (ubuntu 20.04)

### ISSUE THREAD

**Issue by franfreire** [sysstat/sysstat#264]
Hello.

We have noticed that our custom graphs shows an impossible disk write use, when migrating to ubuntu 20. So we have checked the issue and we found that the values of wMB/s when using the JSON output doesn't match with the values of default output. I have checked in 3 different systems, all of them with ubuntu 20 and the result is the same, the  wMB/s is very high when we use the JSON format.

These are the steps to reproduce the issue. We use the command ` iostat /dev/sdb -dmx 1  -o JSON` in one terminal and the same command whithou -o JSON in another one, and you'll see that all the values match except wMB/s, I attach three examples below:

```

"disk": [
	{"disk_device": "nvme0n1", "r/s": 0.00, "w/s": 31.00, "d/s": 0.00, "rMB/s": 0.00, "wMB/s": 248.00, "dMB/s": 0.00, "rrqm/s": 0.00, "wrqm/s": 0.00, "drqm/s": 0.00, "rrqm": 0.00, "wrqm": 0.00, "drqm": 0.00, "r_await": 0.00, "w_await": 1.87, "d_await": 0.00, "rareq-sz": 0.00, "wareq-sz": 4.00, "dareq-sz": 0.00, "aqu-sz": 0.00, "util": 0.80}
	]

Device            r/s     rMB/s   rrqm/s  %rrqm r_await rareq-sz     w/s     wMB/s   wrqm/s  %wrqm w_await wareq-sz     d/s     dMB/s   drqm/s  %drqm d_await dareq-sz  aqu-sz  %util
nvme0n1          0,00      0,00     0,00   0,00    0,00     0,00   31,00      0,12     0,00   0,00    1,87     4,00    0,00      0,00     0,00   0,00    0,00     0,00    0,00   0,80
```

```


"disk": [
	{"disk_device": "sdc", "r/s": 0.00, "w/s": 2.00, "d/s": 0.00, "rMB/s": 0.00, "wMB/s": 24.00, "dMB/s": 0.00, "rrqm/s": 0.00, "wrqm/s": 1.00, "drqm/s": 0.00, "rrqm": 0.00, "wrqm": 33.33, "drqm": 0.00, "r_await": 0.00, "w_await": 2.00, "d_await": 0.00, "rareq-sz": 0.00, "wareq-sz": 6.00, "dareq-sz": 0.00, "aqu-sz": 0.00, "util": 0.40}
	]

Device            r/s     rMB/s   rrqm/s  %rrqm r_await rareq-sz     w/s     wMB/s   wrqm/s  %wrqm w_await wareq-sz     d/s     dMB/s   drqm/s  %drqm d_await dareq-sz  aqu-sz  %util
sdc              0.00      0.00     0.00   0.00    0.00     0.00    2.00      0.01     1.00  33.33    2.00     6.00    0.00      0.00     0.00   0.00    0.00     0.00    0.00   0.40

```
```

"disk": [
	{"disk_device": "sdb", "r/s": 0.00, "w/s": 35.00, "d/s": 0.00, "rMB/s": 0.00, "wMB/s": 280.00, "dMB/s": 0.00, "rrqm/s": 0.00, "wrqm/s": 0.00, "drqm/s": 0.00, "rrqm": 0.00, "wrqm": 0.00, "drqm": 0.00, "r_await": 0.00, "w_await": 0.11, "d_await": 0.00, "rareq-sz": 0.00, "wareq-sz": 4.00, "dareq-sz": 0.00, "aqu-sz": 0.00, "util": 0.40}
	]

Device            r/s     rMB/s   rrqm/s  %rrqm r_await rareq-sz     w/s     wMB/s   wrqm/s  %wrqm w_await wareq-sz     d/s     dMB/s   drqm/s  %drqm d_await dareq-sz  aqu-sz  %util
sdb              0.00      0.00     0.00   0.00    0.00     0.00   35.00      0.14     0.00   0.00    0.11     4.00    0.00      0.00     0.00   0.00    0.00     0.00    0.00   0.40

```

**Comment by sysstat** [comment]
You have just forgotten to indicate which version of sysstat you are using... :-)
FYI commit 404eee1 fixed a bug in iostat where the amount of data read, written or discarded was always expressed in blocks/s (instead of kB or MB depending on the selected unit) in the JSON output. This fix was included in sysstat 12.3.1.

**Comment by franfreire** [comment]
Hello.

Sysstat version on Ubuntu 20 is 12.2.0, I've just compiled the latest version and I've checked that it works properly. Thanks so much for your help. I will close the issue.
